MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -522,8 +524,9 @@ - (void)checkVideoData {
         if (_videoURL) {
             [videoData setVideoSourceUrl:_videoURL];
         }
-        if (_lastAdvertisedBitrate > 0) {
+        if (_lastAdvertisedBitrate > 0 && _started) {
             [videoData setVideoSourceAdvertisedBitrate:@(_lastAdvertisedBitrate)];
+            _lastDispatchedAdvertisedBitrate = _lastAdvertisedBitrate;
```

Comment:
Why did we move this?

Reply:
I moved this back, this needs to be done in the code where we check if those two are the same value to avoid doing unnecessary events.